sklearn/pipeline.py

```diff
@@ -1671,8 +1688,19 @@ def fit_transform(self, X, y=None, **fit_params):
         y : array-like of shape (n_samples, n_outputs), default=None
             Targets for supervised learning.
 
-        **fit_params : dict, default=None
-            Parameters to pass to the fit method of the estimator.
+        **params : dict, default=None
+            - If `enable_metadata_routing=False` (default):
+              Parameters directly passed to the `fit` methods of the
+              transformers.
+
+            - If `enable_metadata_routing=True`:
+              Parameters safely routed to the `fit` methods of the
+              transformers. See :ref:`Metadata Routing User Guide
+              <metadata_routing>` for more details.
+
+            .. versionchanged:: 1.5
+                `**fit_params` are renamed into `**params` and can now be routed via
+                metadata routing API.
```

Comment:
we don't need to mention the rename, since it has no effect for the user's code.